src/sem.c

```diff
@@ -2245,11 +2245,33 @@ static void sem_check_static_elab(tree_t t)
    }
 }
 
+static bool sem_check_missing_wait(tree_t t, nametab_t *tab) {
```

Comment:
```suggestion
static void sem_check_missing_wait(tree_t t, nametab_t *tab) 
{
```

The `return false` is used to indicate an error and sometimes skips subsequent checks, but we don't want that for just a warning so make it `void` instead. Also the `{` goes on a new line for functions.